### DISCUSSION TITLE
Make Select smaller (width)

### DISCUSSION THREAD

**Discussion by jerebtw** [mantinedev/mantine#583] [Q&A] [answered]
How can I make Select smaller without hardcoding the width

For example
![image](https://user-images.githubusercontent.com/49494752/147383425-6dc9cd11-b75d-4c6c-81cd-4cf65e011362.png)

**Comment by rtivital** [comment, marked as answer]
You can set max-width – https://codesandbox.io/s/mantine-template-9ze89?file=/src/App.tsx

**Reply by jerebtw** [reply]
@rtivital Thanks

You updated the default template

**Reply by chrismochinski** [reply]
@rtivital I'm also chasing this option and that template appears to be the default. In fact - I'd love a way to fit the select width to the placeholder text.

**Reply by wallstead** [reply]
The linked solution doesn't load for me, just shows a "Your app here". Really wish I could see the max width solution mentioned.

**Reply by jerebtw** [reply]
https://codesandbox.io/s/currying-wave-lktccw

**Reply by shindavid** [reply]
@jerebtw Sandbox not found

**Reply by jerebtw** [reply]
This should be the answer https://codesandbox.io/s/exciting-danilo-qpz72t

**Reply by rafpaf** [reply]
@jerebtw The sandbox [codesandbox.io/s/exciting-danilo-qpz72t](https://codesandbox.io/s/exciting-danilo-qpz72t) is no longer available

**Reply by jerebtw** [reply]
@rafpaf you can just set the `maw` prop like:

![image](https://github.com/mantinedev/mantine/assets/49494752/b7dca1db-994b-497a-a118-41326b3cd59c)
